servers/src/main/java/tachyon/worker/block/evictor/LRUEvictor.java

```diff
@@ -99,20 +126,12 @@ public EvictionPlan freeSpace(long availableBytes, BlockStoreLocation location)
    */
   @Override
   public void onAccessBlock(long userId, long blockId) {
-    if (mLRUCache.containsKey(blockId)) {
-      mLRUCache.get(blockId);
-    } else {
-      mLRUCache.put(blockId, true);
-    }
+    mLRUCache.put(blockId, true);
```

Comment:
Actually, I found this `true` value used multiple times in this class with the same purpose to pass into a map, I suggest to define a boolean const of this value (`true`) in names such as UNUSED_MAP_VALUE or something else making better sense to you, refer to this const value in this class.